README.md

```diff
@@ -39,6 +39,4 @@ The Java client doesn't support managing Pinecone services, only reading and wri
 
 ## Examples
 
-- The most basic example usage is in `src/test/java/io/pinecone/PineconeClientLiveIntegTest.java`, covering most basic operations.
-> [!NOTE]
-> The java-basic-mvn example is outdated.
\ No newline at end of file
+- The data and control plane operations can be found in `io/pinecone/integration` folder.
```

Comment:
Got rid of the note since I have added comments to the examples where the code was outdated.

Reply:
I know we're in the `## Examples` section, but maybe a small tweak to the wording here:

```suggestion
- Examples for the data and control plane operations can be found in `io/pinecone/integration` folder.
```

Also, do we call out the `/examples/` folder at the root of the app anywhere?

Reply:
I'll be updating examples folder later on so I'm not going to call it out for now. Also big changes are coming to README file later on.